### DISCUSSION TITLE
Gamepad Support Ever Coming?

### DISCUSSION THREAD

**Discussion by seanocali** [microsoft/WindowsAppSDK#1608] [Q&A] [answered]
I have a regular Xbox One controller plugged into my PC. Button presses do not register a Keydown event like they do in UWP. `Windows.Gaming.Input.Gamepad.Gamepads` is empty. 

I remember an earlier version of the roadmap document stating that gamepad support would be available in 0.8. Testing 1.0 preview 2 right now and it's still not there. 

Any info on when or if this will be supported?

**Comment by ghost1372** [comment, marked as answer]
ask your question here
https://github.com/microsoft/microsoft-ui-xaml/issues/6050
